tests/vmi_lifecycle_test.go

```diff
@@ -485,6 +485,37 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 			})
 		})
 
+		Context("when guest crashes", Label("crashing-guest"), decorators.VMIlifecycle, func() {
```

Comment:
Why the label?

Reply:
Sorry this was meant only for my testing. Will remove it